sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 - Replaced static functions from `KeyOperation` and `KeyCurveName` with static const members.
 - Replaced the enum `JsonWebKeyType` for a class with static const members as an extensible enum called `KeyVaultKeyType`.
 - Renamed `MaxResults` to `MaxPageResults` for `GetSinglePageOptions`.
+- Changed the returned type from list keys, key versions and deleted keys from `Response<T>` to `PagedResponse<T>` affecting:
+  - `GetPropertiesOfKeysSinglePage` and `GetPropertiesOfKeyVersionsSinglePage` now returns `KeyPropertiesSinglePage`.
+  - `GetDeletedKeysSinglePage` now returns `DeletedKeySinglePage`.
```

Comment:
We need to rename these methods to be more user-friendly, removing the `SinglePage` pattern.
It's fine if you want to defer doing that separately from this PR.

```suggestion
  - `GetDeletedKeysSinglePage()` now returns `DeletedKeySinglePage`.
```